.github/workflows/CI.yml

```diff
@@ -0,0 +1,60 @@
+name: CI
+
+on:
+  pull_request:
+    paths:
+      - '**.scala'
+      - '**.java'
+      - '**.sbt'
+      - '.github/workflows/**.yml'
+  push:
+    branches:
+      - master
+    paths:
+      - '**.scala'
+      - '**.java'
+      - '**.sbt'
+      - '.github/workflows/**.yml'
+
+jobs:
+  code_format:
+    name: Code Format
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: jcheckstyle
+        run: ./sbt jcheckStyle
+  test_jdk11:
+    name: Test JDK11
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-scala@v7
```

Comment:
If the AdoptJDK installation is supported in setup-java action, it's better to replace with it.
https://github.com/actions/setup-java

Reply:
msgpack-java's test is written in Scala, and also setup-scala can install natively-built sbt (csbt) using GraalVM. So let me keep using setup-scala.